docling_serve/response_preparation.py

```diff
@@ -234,9 +246,40 @@ def process_results(
         # Output directory
         # background_tasks.add_task(shutil.rmtree, work_dir, ignore_errors=True)
 
-        response = FileResponse(
-            file_path, filename=file_path.name, media_type="application/zip"
-        )
+        if isinstance(target, PutTarget):
```

Comment:
`PutTarget` should already be a presigned URL provided by the user. Then in the block below we should simply use `httpx.put`